osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderTail.cs

```diff
@@ -12,6 +14,8 @@ public class DrawableSliderTail : DrawableOsuHitObject, IRequireTracking
     {
         private readonly Slider slider;
 
+        private readonly IBindable<float> scaleBindable = new Bindable<float>();
```

Comment:
Just `scale` would do. The type will tell the rest.

Reply:
I assume this is just for consistency with `poisitionBindable` and the naming of bindables inside the hitobject itself. I think a better name might be `hitObjectScale` and `hitObjectPosition`, just so that it doesn't conflict with the drawable's own scale/position.